app/views/request/_sidebar.html.erb

```diff
@@ -15,44 +15,25 @@
     <%= render :partial => 'track/rss_feed', :locals => { :track_thing => @track_thing, :location => 'sidebar' } %>
   </div>
 
-  <% if @info_request.described_state != "attention_requested" %>
```

Comment:
I think this can be removed - we could add a more info link next to the report link in the action menu, and this sort of thing can be in the help docs